internals/compiler.rst

```diff
@@ -579,18 +580,14 @@ Important Files
 
    * :cpy-file:`Python/symtable.c`: Generates a symbol table from AST.
 
-   * :cpy-file:`Python/peephole.c`: Optimizes the bytecode.
-
    * :cpy-file:`Python/pyarena.c`: Implementation of the arena memory manager.
 
-   * :cpy-file:`Python/wordcode_helpers.h`: Helpers for generating bytecode.
```

Comment:
Both got merged into `Python/compile.c`, which is already listed above, so I removed these.